app/controllers/api/v0/admins_controller.rb

```diff
@@ -29,6 +29,8 @@ def add_portfolio_item_to_portfolio
       def add_portfolio_item
         so = ServiceOffering::AddToPortfolioItem.new(portfolio_item_params)
         render :json => so.process.item
+      rescue ServiceCatalog::TopologyError => e
```

Comment:
Small nit, but wouldn't mind seeing specs around this exception.

Reply:
Added!